src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -678,4 +678,20 @@ object messages {
            |$fixedVarInAlternative""".stripMargin
     }
   }
+
+  case class IncorrectRepeatedParameterSyntax()(implicit ctx: Context) extends Message(24) {
+    val kind = "Syntax"
+
+    val msg = "'*' expected"
+
+    val explanation =
+      hl"""|Expected * in '_*' operator.
+           |
+           |If you were to have a method such as:
+           |${"def func(args: Int*) = _ // do something"}
+           |
+           |Then you would provide arguments to it as such:
+           |${"func(Seq(1, 2, 3): _*)"}
+           |""".stripMargin
+  }
```

Comment:
This can also happen in:

```scala
case class Foo(i: Int*)

(null: Foo) match {
  case Foo(i: _) => // ...
}
```

And providing `func` with arguments could also be done like: `func(1, 2, 3)` right? Basically I think your explanation is good, but try to explain what the `_*` construct does :)